community/config/eggs/creating_a_custom_egg.md

```diff
@@ -201,3 +201,20 @@ regex as any letters or numbers (`\w\d`) including underscore (`_`), periods (`.
 They will then be visible when managing the startup for a server in both the Admin CP and on the Front-End.
 
 ![](../../../.vuepress/public/community/config/eggs/Pterodactyl_Create_New_Egg_Startup.png)
+
+## List of default variables
+
+The default variables are always accessible to all eggs and don't have to be created separately. They can be used in the egg startup, install script, or the configuration file parser.
+
+| Variable | Description | Example |
+|----------|-------------|---------|
+| TZ       | Time Zone set from panels `.env` |  `Etc/UTC` |
+| STARTUP  | Startup command of the egg | `./Process` |
```

Comment:
Might be good to use an actual line from the vanilla MC server or something showing that the variables aren't replaced in this yet.